var/spack/repos/builtin/packages/rpp/package.py

```diff
@@ -152,7 +153,7 @@ def patch(self):
 
     with when("+hip"):
         with when("@5.7:"):
-            for ver in ["5.7.0", "5.7.1", "6.0.0", "6.0.2", "6.1.0", "6.1.1", "6.1.2"]:
+            for ver in ["5.7.0", "5.7.1", "6.0.0", "6.0.2", "6.1.0", "6.1.1", "6.1.2", "6.2.0"]:
```

Comment:
can you please have the new version of boost . depends_on("boost@1.72.0:1.85.0") in the rpp package.